### ISSUE TITLE
large errors with linear quantized op on MPS device

### ISSUE THREAD

**Issue by dacorvo** [huggingface/quanto#13]
Some of the dispatched `linear` op tests are failing with large quantization errors on `mps` device.

```
FAILED test/nn/test_qlinear.py::test_quantize_linear[mps-bias-32-32-1] - AssertionError: assert tensor(127, device='mps:0', dtype=torch.int8) <= 1
FAILED test/nn/test_qlinear.py::test_quantize_custom_module - assert tensor(2, dtype=torch.int8) <= 1
FAILED test/qtensor/test_quantized_dispatch.py::test_linear[mps-bias-5-5-1] - AssertionError: error 3.1107 exceeds atol 0.0001
FAILED test/qtensor/test_quantized_dispatch.py::test_linear[mps-bias-32-32-1] - AssertionError: error 9.2124 exceeds atol 0.0001
```

**Comment by dacorvo** [comment]
This only happens when there is a bias, so there must be something wrong related to the `torch.ops.aten.linear` MPS implementation. Maybe check for updates to references instead of copies that are hidden by other backends ?

**Comment by dacorvo** [comment]
Resolved by #37
